services/app/apps/codebattle/test/codebattle/utils/populate_users_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Codebattle.Utils.PopulateUsersTest do
+  use Codebattle.DataCase, async: true
+  import Mock
+
+  test "from_csv" do
+    csv = """
+    name,password
+    user1,p@ssw0rd!
+    user2,adminadmin1234
+    user3,changem3
+    user4,"hop hey lala ley"
+    """
+
+    {fd, path} = Temp.open!()
+    IO.write(fd, csv)
+    File.close(fd)
+
+    assert {4, nil} = Codebattle.Utils.PopulateUsers.from_csv(path)
```

Comment:
let's add check that `Codebattle.User.authenticate(name, password)` 
wors for the first user